server/routes/home.js

```diff
@@ -2,9 +2,9 @@
 
 const router = require('express').Router();
 
-const homeValidator = require('../home/homeValidator');
+
 const homeController = require('../home/homeController');
 
-router.get('/homePage', homeController.getHomeInfo); //Gets information for home page
+router.get('/homePage?', homeController.getHomeInfo); //Gets information for home page
```

Comment:
What does the ? do here?

Reply:
the ? allows for me to just split the string when it comes into server easy.  so i can use string.split('?')[1] - to target the id